src/widgets/games/speed-counting/ui/speed-counting-game.vue

```diff
@@ -1,15 +1,38 @@
 <script setup lang="ts">
+import {
+  computed,
+  watchEffect,
+} from 'vue';
 import { useSavedGames } from '@/features/saved-games';
 import {
   formatTime,
   useStopwatch,
 } from '@/features/stopwatch';
-import {
-  GameCard,
-  useGameStore,
-} from '@/entities/game';
+import GameResults from './game-results.vue';
+import PlayGame from './play-game.vue';
+import StartGame from './start-game.vue';
+
+interface Props {
+  isNotStarted: boolean;
+  isStarted: boolean;
+  isPaused: boolean;
+  isResumed: boolean;
+  isShowingResults: boolean;
+}
+type Emits = (event: 'start' | 'showResult') => void;
+
+const props = defineProps<Props>();
+const emit = defineEmits<Emits>();
+watchEffect(() => {
+  if (props.isPaused) {
+    stopStopwatch();
+  } else if (props.isResumed) {
+    startStopwatch();
+  }
+});
```

Comment:
Use `defineExpose` to provide to the parent component (game-layout) 2 methods: `play` and `pause`. Then u can use it once at layout and next, when u will add new games, u will only need to provide these two methods again without writing duplicated logic